packages/utils/src/lib/reports/generate-md-report.unit.test.ts

```diff
@@ -135,31 +135,39 @@ describe('auditDetailsIssues', () => {
   it('should include source file', () => {
     expect(
       auditDetailsIssues([
-        { source: { file: 'index.js' }, severity: 'error' } as Issue,
+        { source: { file: '/index.js' }, severity: 'error' } as Issue,
       ])?.toString(),
-    ).toMatch('`index.js`');
+    ).toMatch('`/index.js`');
```

Comment:
All issue file paths will be relative to git root directory at this point, so won't start with `/`. The core [converts all issues source files](https://github.com/code-pushup/cli/blob/main/packages/core/src/lib/normalize.ts#L13) right after [plugin execution](https://github.com/code-pushup/cli/blob/main/packages/core/src/lib/implementation/execute-plugin.ts#L85).

So realistic examples would be `index.js` or `src/index.js`.